frameworks/elastic/tests/test_rack_awareness.py

```diff
@@ -1,6 +1,7 @@
 import logging
-
 import pytest
+from toolz import get_in
```

Comment:
Does this require a change to some `requirements.txt` file?

Reply:
[Yea](https://github.com/mesosphere/dcos-commons/pull/2680/files#diff-bdeaf6bb06566ad0d723cd60c75e563dR11).

Reply:
Sorry, I missed it.

Reply:
Are you opposed to `import toolz` and then using `toolz.get_in`? The same number of lines will be changed.

Reply:
(as with `merge_dictionaries` we could also call the `toolz` implementation from `sdk_utils` instead.